frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -185,101 +245,81 @@ const ConsolePluginsTable: React.FC<ConsolePluginsTableProps> = ({ obj, rows, lo
+                <Tr key={name}>
+                  <Td dataLabel={columns[0].id}>
+                    <ResourceLink groupVersionKind={consolePluginGVK} name={name} hideIcon />
+                  </Td>
+                  <Td dataLabel={columns[1].id}>{version || DASH}</Td>
+                  <Td dataLabel={columns[2].id}>{description || DASH}</Td>
+                  <Td dataLabel={columns[3].id}>
+                    <Status
+                      status={status}
+                      title={status === 'Failed' ? errorMessage : undefined}
+                    />
+                  </Td>
+                  <Td dataLabel={columns[4].id}>
+                    <ConsolePluginEnabledStatus pluginName={name} enabled={enabled} />
+                  </Td>
+                  <Td dataLabel={columns[5].id}>
+                    <ConsolePluginCSPStatus hasViolations={hasCSPViolations ?? false} />
+                  </Td>
+                </Tr>
+              ),
+            )}
           </Tbody>
         </Table>
       ) : (
-        <EmptyBox label={t('console-app~console plugins')} />
+        <EmptyBox label={t('console-app~Console plugins')} />
       )}
     </div>
   );
 };
 
 const DevPluginsPage: React.FCC<ConsoleOperatorConfigPageProps> = (props) => {
-  const [pluginInfo, pluginInfoLoaded] = useDynamicPluginInfo();
+  const [pluginInfoEntries, allPluginsProcessed] = useDynamicPluginInfo();
+
   const rows = React.useMemo<ConsolePluginTableRow[]>(
-    () =>
-      !pluginInfoLoaded
-        ? []
-        : pluginInfo.filter(isLoadedDynamicPluginInfo).map((plugin) => ({
-            name: plugin.metadata.name,
-            version: plugin.metadata.version,
-            description: plugin.metadata?.customProperties?.console?.description,
-            enabled: plugin.enabled,
-            status: plugin.status,
-          })),
-    [pluginInfo, pluginInfoLoaded],
+    () => (allPluginsProcessed ? pluginInfoEntries.map(getConsolePluginDisplayData) : []),
+    [pluginInfoEntries, allPluginsProcessed],
   );
-  return <ConsolePluginsTable {...props} rows={rows} loaded={pluginInfoLoaded} />;
+
+  return <ConsolePluginsTable {...props} rows={rows} loaded={allPluginsProcessed} />;
 };
 
 const PluginsPage: React.FC<ConsoleOperatorConfigPageProps> = (props) => {
-  const [pluginInfo, pluginInfoLoaded] = useDynamicPluginInfo();
-  const [consolePlugins, consolePluginsLoaded] = useK8sWatchResource<ConsolePluginKind[]>({
-    isList: true,
-    kind: referenceForModel(ConsolePluginModel),
-  });
+  const [pluginInfoEntries, allPluginsProcessed] = useDynamicPluginInfo();
+
   const enabledPlugins = React.useMemo(() => props?.obj?.spec?.plugins ?? [], [
     props?.obj?.spec?.plugins,
   ]);
+
   const rows = React.useMemo<ConsolePluginTableRow[]>(() => {
-    if (!pluginInfoLoaded || !consolePluginsLoaded) {
+    if (!allPluginsProcessed) {
       return [];
     }
-    return consolePlugins.map((plugin) => {
```

Comment:
@vojtechszocs, we want this existing code since `pluginInfoEntries` only returns plugins in the pluginStore.

Reply:
Confirmed restoring `<PluginsPage>` to this state fixes the bug.
![console-openshift-console apps rhamilto devcluster openshift com_k8s_cluster_operator openshift io~v1~Console_cluster_console-plugins](https://github.com/user-attachments/assets/303afe9a-73b5-485e-aedc-8982cf9e7175)

Reply:
Here is a patch that reverts that changes to `<DevPluginsPage>` and `<PluginsPage>`.

[0001-Fix-for-https-github.com-openshift-console-pull-1437.patch](https://github.com/user-attachments/files/17757624/0001-Fix-for-https-github.com-openshift-console-pull-1437.patch)